app/models/volunteer.rb

```diff
@@ -143,6 +173,10 @@ def to_s
 
   private
 
+  def record_acceptance_changed
+    self["#{acceptance_change[1]}_at".to_sym] = Time.zone.now if acceptance_changed?
```

Comment:
`acceptance` should do the same as `acceptance_change[1]` here and would be more obvious :)

Reply:
Makes sense.